Documentation/workflow/UsingYourBuild.md

```diff
@@ -44,22 +44,24 @@ dotnet new console
 
 ### Change project to be self-contained
 
-In order to update with your local changes the application needs to be self-contained, as opposed to running on the
+In order to update with your local changes, the application needs to be self-contained, as opposed to running on the
 shared framework.  In order to do that you will need to add a `RuntimeIdentifier` to your project.
 
-```
-  <PropertyGroup>
-    ...
-    <RuntimeIdentifier>win-x64</RuntimeIdentifier>
-  </PropertyGroup>
+```xml
+<PropertyGroup>
+  ...
+  <RuntimeIdentifier>win-x64</RuntimeIdentifier>
+</PropertyGroup>
 ```
 
 For windows you will want `win-x64` but for other OS's you will need to set it to the most appropriate one based
 on what you built. You can generally figure that out by looking at the packages you found in your output. In our
 example you will see there is a package with the name `runtime.win-x64.Microsoft.NETCore.Runtime.CoreCLR.2.1.0-beta-25023-0.nupkg`
 so you will want to put whatever id is between `runtime.` and `Microsoft.NETCore.Runtime.CoreCLR`.
```

Comment:
I think you still need to point nuget to `https://dotnet.myget.org/F/dotnet-core/api/v3/index.json` using NuGet.config or alternative. It won't restore without it.